cspell.json

```diff
@@ -336,7 +336,8 @@
         "Autoloader",
         "autoloader",
         "qwertyuiopasdfghjklzxccvbnm",
-        "QWERTYUIOPASDFGHJKLZXCVBNM"
+        "QWERTYUIOPASDFGHJKLZXCVBNM",
+        "Servicecentovacast"
```

Comment:
Servicecentovacast has been dropped so why not remove it from 

https://github.com/FOSSBilling/FOSSBilling/pull/1212/files#diff-8ec5bae185dc9af5b5bfdcd5f1dcbee62f18525ee488996d52b22729f4076ef0